session_rpcserver.go

```diff
@@ -11,16 +11,18 @@ import (
 	"github.com/lightninglabs/lightning-node-connect/mailbox"
 	"github.com/lightninglabs/lightning-terminal/litrpc"
 	"github.com/lightninglabs/lightning-terminal/session"
+	"github.com/lightninglabs/lndclient"
 	"google.golang.org/grpc"
 )
 
 // sessionRpcServer is the gRPC server for the Session RPC interface.
 type sessionRpcServer struct {
 	litrpc.UnimplementedSessionsServer
 
-	cfg           *sessionRpcServerConfig
-	db            *session.DB
-	sessionServer *session.Server
+	cfg             *sessionRpcServerConfig
+	db              *session.DB
+	sessionServer   *session.Server
+	macaroonService *lndclient.MacaroonService
```

Comment:
After starting with the accounts feature, I think it would make sense to keep the macaroon service as part of the main terminal struct itself. Because the service will also validate macaroons for the accounts RPC service.

Reply:
cool, will move it out 👍